README.md

```diff
@@ -13,61 +13,118 @@
+[recording rules](https://docs.timescale.com/promscale/latest/downsample-data/recording/).
+
+**Key features**: 100% PromQL-compliant, high availability, multi-tenancy, PromQL alerting and recording rules, per-metric retention.
+
+If you use Prometheus and are also familiar with PostgreSQL, then Promscale is a great choice for
+your Prometheus remote storage. You can scale to millions of series and hundreds of thousands of samples
+on a single PostgreSQL node thanks to TimescaleDB.
+
+To get started:
+1. [Install Promscale](https://docs.timescale.com/promscale/latest/installation/#install-promscale-with-instrumentation).
+2. [Configure Prometheus](https://docs.timescale.com/promscale/latest/send-data/prometheus/) to send data to Promscale.
+3. [Configure Grafana](https://docs.timescale.com/promscale/latest/visualize-data/grafana/) to query and visualize metrics from Promscale
+using a PromQL and/or a PostgreSQL datasource.
+
+## Promscale for Jaeger and OpenTelemetry
+
+Promscale can natively ingest OpenTelemetry traces and it can also ingest Jaeger and Zipkin traces via
+the OpenTelemetry Collector.
+
+Promscale solves three main use cases for Jaeger and OpenTelemetry users:
+
+* **Easy-to-use durable and scalable storage backend for traces**<br/>
+Most users run Jaeger with the in memory or badger storage because the two options for a more durable storage
+(Elasticsearch and Cassandra) are difficult to set up and operate. Promscale uses a much simpler architecture
+based on PostgreSQL which many developers are comfortable with and scales to 100s of thousands of spans per
+second on a single database node.
+
+* **Service performance analysis**<br/>
+Because Promscale can store both metrics and traces, you can use the new 
+[Service Performance Management](https://www.jaegertracing.io/docs/1.36/spm/) feature in Jaeger with Promscale 
+as the only storage backend for the entire experience.
+Promscale also includes a fully customizable, out-of-the-box, and modern
+[Application Performance Management (APM) experience](https://docs.timescale.com/promscale/latest/visualize-data/apm-experience/)
+in Grafana built using SQL queries on traces.
 
-For a detailed description of the initial architecture of Promscale for Prometheus metrics that covers
-some key design principles, please see [our design doc](https://tsdb.co/prom-design-doc).
+* **Trace analysis**<br/>
+Jaeger searching capabilities are limited to filtering individual traces. This is helpful when troubleshooting problems once you know
+what you are looking for. With Promscale you can use SQL to interrogate your trace data in any way you want and discover issues
+that would usally take you a long time to figure out by just looking at log lines, metric charts or individual traces. You can see some
+examples in [this blog post](https://www.timescale.com/blog/what-are-traces-and-how-sql-yes-sql-and-opentelemetry-can-help-us-get-more-value-out-of-traces-to-build-better-software/)
```

Comment:
Could we put a simple example here? At least just the SQL explaining what we are querying. I think this is a key point of the power of the product, and having to read an entire blogpost might not be very friendly.

Maybe something like:

```
SELECT
    client_span.service_name AS client_service,
    server_span.service_name AS server_service,
    server_span.span_name AS server_operation,
    count(*) AS number_of_requests,
    ROUND(sum(server_span.duration_ms)::numeric) AS total_exec_time
FROM
    span AS server_span
    JOIN span AS client_span
    ON server_span.parent_span_id = client_span.span_id
WHERE
    client_span.start_time > NOW() - INTERVAL '30 minutes' AND
    client_span.service_name != server_span.service_name
GROUP BY
    client_span.service_name,
    server_span.service_name,
    server_span.span_name
ORDER BY
    server_service,
    server_operation,
    number_of_requests DESC;
```
    
> In this query we are leveraging another powerful capability of SQL: joins. We are joining the span table with itself to identify only the spans that represent a call between two services. 

I think even though it might not make total sense on a first glance it might give an idea on the power of using SQL. If we could show (in a short paragraph) how cumbersome it would be to get the same information without SQL would be plus.

Reply:
I am not sure the readme of the repo is the place to put this. I wanted to put folks to our documentation for querying traces which also has examples but we it's still being reviewed.

Reply:
Sounds good, I think I just wanted to give people reading this a glimpse of what they can achieve by using SQL to query the telemetry data.